src/main/java/uk/ac/ebi/eva/pipeline/parameters/InputParameters.java

```diff
@@ -63,7 +63,6 @@
     @Value(PARAMETER + JobParametersNames.CONFIG_CHUNK_SIZE + "']?:1000}")
     private int chunkSize;
```

Comment:
I agree with the comment that the last 3 parameters should be moved to a different class. Another PR?